### ISSUE TITLE
ui-switch inside a ui-tab component has an invisible track

### ISSUE THREAD

**Issue by andrei-pmbcn** [JosephusPaye/Keen-UI#392]
Try the following:
```
<ui-tabs type="text">
  <ui-tab title="test">
    <ui-switch></ui-switch>
  </ui-tab>
</ui-tabs>
```

The switch's track will be invisible, making the switch awkward to use. This is because the css background-color in ui-tabs__body is set to white. Disabling the background-color line in ui-tabs__body makes the switch's track visible again.

**Comment by JosephusPaye** [comment]
Hey @andrei-pmbcn,

It seems like a `z-index` issue. Can be fixed using the following CSS:

```css
.ui-switch__track {
    z-index: initial;
}

.ui-switch__thumb {
    z-index: 1;
}
```

See here for a demo: https://codepen.io/JosephusPaye/pen/MQBvbM.

This will be fixed in the next version.

**Comment by liammcsheffrey** [comment]
Hi @JosephusPaye - really loving the library! Do you have an estimated date for the next version?

**Comment by JosephusPaye** [comment]
Fixed in https://github.com/JosephusPaye/Keen-UI/commit/b5fc17d7ff0627470f9547db288209cce5cdd748.

**Comment by JosephusPaye** [comment]
Fix released in [v1.1.0](https://github.com/JosephusPaye/Keen-UI/releases/tag/v1.1.0).
